MatrixSDK/MXRestClient.m

```diff
@@ -3200,7 +3228,7 @@ - (MXHTTPOperation*)roomSummaryWith:(NSString*)roomIdOrAlias
                             success:(void (^)(MXPublicRoom *room))success
                             failure:(void (^)(NSError *error))failure
 {
-    NSMutableString *path = [NSMutableString stringWithFormat:@"%@/im.nheko.summary/rooms/%@/summary", kMXAPIPrefixPathUnstable, roomIdOrAlias];
+    NSMutableString *path = [NSMutableString stringWithFormat:@"%@/im.nheko.summary/rooms/%@/summary", kMXAPIPrefixPathUnstable, [MXTools encodeURIComponent:roomIdOrAlias]];
```

Comment:
Looks sensible, but not sure I understand how this change relates to the MSC.

Reply:
Indeed. I've just added this code to fix some eventual issues. Do you want me to revert it?

Reply:
This could definitely happen to me :D